interface/main/holidays/import_holidays.php

```diff
@@ -0,0 +1,177 @@
+<?php
+
+//this part download the CSV file after the click on the href link
+if($_GET['download_file']==1)
+{
+    if ( ! file_exists($_GET['filename']))
+    {
+        echo 'file missing';
+    }
+    else
+    {
+        header('HTTP/1.1 200 OK');
+        header('Cache-Control: no-cache, must-revalidate');
+        header("Pragma: no-cache");
+        header("Expires: 0");
+        header("Content-type: text/csv");
+        header("Content-Disposition: attachment; filename='holiday.csv'");
+        readfile($_GET['filename']);
+        exit;
+    }
+    die();
+}
+// end download section
+
+?>
+<?php
+/**
+ *
+ * @package OpenEMR
+ * @author  sharon cohen <sharonco@matrix.co.il>
+ * @link    http://www.open-emr.org
+ */
```

Comment:
When downloading a file we need to set the header so it does the download.
Do you see any problem with it or do you have any suggestion?